docs/software/comanage-troubleshooting-guide.md

```diff
@@ -0,0 +1,50 @@
+COManage Troubleshooting Guide
+==============================
+
+A resource for COManage administrators to find solutions to commonly encountered problems.
+
+
+COManage Troubleshooting Items
+----
+This section contains some of the issues you may encounter when interacting with the OSG COManage.
+
+Problem list:
+
+ - [Valid Tiger User is unauthorized with Dex credentials](#valid-tiger-user-is-unauthorized-with-dex-credentials)
```

Comment:
This is unnecessary since we use MkDocs here and it will generate a table of contents for us based on the headings